pouchdb-core/pouchdb-core-tests.ts

```diff
@@ -40,9 +40,9 @@ namespace PouchDBCoreTests {
 
     function testBulkDocs() {
         const db = new PouchDB<MyModel>();
-        type MyModel = { property: 'someProperty '};
-        let model: PouchDB.Core.Document<MyModel>;
```

Comment:
Yeah, but i thing whole purpose of tests is to test Type. See testChanges(), I saw it in jquery test or somewhere. Updating definitions needs also updating tests.